sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -302,8 +302,14 @@ functionCall
     | relevanceFunction                                             #relevanceFunctionCall
     | highlightFunction                                             #highlightFunctionCall
     | constantFunction                                              #constantFunctionCall
+    | nestedFunction                                                #nestedFunctionCall
     ;
 
+nestedFunction
+    : NESTED LR_BRACKET nestedField RR_BRACKET
```

Comment:
Does this support the following query?
`SELECT nested(message.info) FROM multi_nested WHERE nested('message', message.info = 'a' OR message.info = 'a')`

Specifically the second part 
`nested('message', message.info = 'a' OR message.info = 'a')`

Reply:
It does not. Nested in this PoC is only supported in SELECT. Nested in WHERE will be implemented in follow up tasks.